src/motor/analog/pwm/BrushedMotor.hpp

```diff
@@ -33,9 +33,9 @@ struct BrushedMotorPins
     {
     }
 
-    const uint8_t forward;  // NOLINT: It's OK for these to be public
-    const uint8_t backward; // NOLINT
-    const uint8_t enable;   // NOLINT
+    const uint8_t forward;
```

Comment:
:warning: **misc-non-private-member-variables-in-classes** :warning:
member variable 'forward' has public visibility